useragent/runtime_test.go

```diff
@@ -0,0 +1,26 @@
+package useragent
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUserAgentRuntime(t *testing.T) {
+	for _, tc := range []string{
+		"1.2.3",
+		"0.0.0-dev+2e014739024a",
+		"client.0",
+		"foo",
+		"15.0",
+		"13.3",
```

Comment:
Can you include an emoji/invalid character here as well?

I think this test passes if you remove the `Sanitize` call from the function.

Reply:
done